projects/Cura-Doodle3D.cmake

```diff
@@ -0,0 +1,8 @@
+
+ExternalProject_Add(Cura-Doodle3D
+    GIT_REPOSITORY https://github.com/Doodle3D/Doodle3D-cura-plugin
+    GIT_TAG origin/master
+    CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${EXTERNALPROJECT_INSTALL_PREFIX} -DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}
+)
+
+SetProjectDependencies(TARGET Cura-Doodle3D DEPENDS Cura)
```

Comment:
This file is currently also included in the build, because the glob gets all .cmake files in this directory.